internal/visualization/console/plugin.go

```diff
@@ -10,6 +10,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	korrel8rService   = "korrel8r"
+	korrel8rNamespace = "korrel8r"
```

Comment:
will this indeed be the NS?  Is it something intended to live in openshift?  If it is an observability service, do we have a preferred NS prefix for the observability org?

Reply:
@alanconway don't see this comment addressed..

Reply:
I don't know what the long term story will be. If you prefer 'openshift-korrel8r' as more consistent for now, I'll make the change. There isn't anything concrete from the observability side yet.

Reply:
Going with korrel8r for now, bu taking note that we need to resolve the correct naming before tech preview.